reana_client/cli/files.py

```diff
@@ -140,8 +141,19 @@ def get_files(ctx, workflow, _filter,
     help='Path to the directory where files will be downloaded.')
 @add_access_token_options
 @click.pass_context
-def download_files(ctx, workflow, filenames, output_directory, access_token):
-    """Download workflow workspace file(s)."""
+def download_files(ctx, workflow, filenames,
+                   output_directory, access_token):  # noqa: D301
+    """Download workflow files.
+
+    This command permits to download workflow files from its workspace. If
```

Comment:
This command permits to download workspace files.  By default, the files specified in the workflow specification as outputs are downloaded.  You can also specify the individual files you would like to download, see examples below.  Note that downloading directories is not yet supported.